e2e/tests/overview-page.spec.ts

```diff
@@ -77,7 +77,22 @@ test('shows a tooltip when hovering over a baseline chip', async ({page}) => {
 });
 
 test('Export to CSV button downloads a file', async ({page}) => {
-  await gotoOverviewPageUrl(page, 'http://localhost:5555/');
+  const columnIds = [
```

Comment:
I can add a new test to generate all-columns.  

Since this is a playwright spec, it does not have access to the application code, but only the web browser interface.  So I couldn't import the list of all column ids and use that programmatically.  Having the test open the column selector dialog and select all the checkboxes might be possible, if we can query for a list of those checkboxes.  Seems feasible.